engine/src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -161,7 +178,7 @@ case class SgeBackend(actorSystem: ActorSystem) extends Backend with SharedFileS
   private def launchQsub(jobDescriptor: BackendCallJobDescriptor): (Int, Option[Int]) = {
     val logger = jobLogger(jobDescriptor)
     val sgeJobName = s"cromwell_${jobDescriptor.workflowDescriptor.shortId}_${jobDescriptor.call.unqualifiedName}"
-    val argv = Seq("qsub", "-terse", "-N", sgeJobName, "-V", "-b", "n", "-wd", jobDescriptor.callRootPath.toAbsolutePath, "-o", jobDescriptor.stdout.getFileName, "-e", jobDescriptor.stderr.getFileName, jobDescriptor.script.toAbsolutePath).map(_.toString)
+    val argv = (Seq("qsub","-terse", "-N", sgeJobName, "-V", "-b", "n", "-wd", jobDescriptor.callRootPath.toAbsolutePath) ++ InfoKeys.QueueName.map(Seq("-q",  _)).getOrElse(Seq()) ++ InfoKeys.ProjectName.map(Seq("-P", _)).getOrElse(Seq()) ++ Seq("-o", jobDescriptor.stdout.getFileName, "-e", jobDescriptor.stderr.getFileName, jobDescriptor.script.toAbsolutePath)).map(_.toString)
```

Comment:
IMHO the original line was already too long, how about pulling out the new optional parameters to locals?  Also `Seq.empty` would be preferred for an empty Seq here:

```scala
val queueParam = InfoKeys.QueueName.map(Seq("-q",  _)).getOrElse(Seq.empty[String])
...
val argv = ... ++ queueParam ++ ...
```